compiler/modules/front.syntax/token.bal

```diff
@@ -245,10 +251,18 @@ class Tokenizer {
         self.mode = m;
     }
 
-    function currentPos() returns Position {
+    function currentStartPos() returns Position {
         return createPosition(self.lineIndex, self.tokenStartCodePointIndex);
     }
 
+    function currentEndPos() returns Position {
+        return createPosition(self.lineIndex, self.codePointIndex);
+    }
+
+    function prevEndPos() returns Position {
```

Comment:
To be consistent, this should be `previousEndPos` (or we could have `curStartPos` and `curEndPos`)

Reply:
fixed